packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -309,32 +311,111 @@ tbody.spectrum-Table-body {
 }
 
 .spectrum-Table-colResizeIndicator {
+  --spectrum-table-col-resize-indicator-background-color: var(--spectrum-global-color-blue-600);
```

Comment:
define this (and all others) at the top on the `.spectrum-Table` selector, it will help us remove more selectors from the forced-colors media query section